<?xml version="1.0"?>
<launch>

    <!-- gripper open/close angle configuration -->
    <arg name="gripper_configuration_open" default="-1.5" /> <!-- -1.5 maximum -->
    <arg name="gripper_configuration_close" default="-0.2" /> <!-- -0.2 maximum -->
    <arg name="gripper_joint_name" default="left_arm_gripper_joint" />
    <arg name="gripper_command_topic" default="left_arm_gripper_joint_position_controller/command" />
    <arg name="gripper_subcribe_command_topic" default="left_arm_gripper/gripper_command" />

    <!-- Start joint trajectory action controller for gripper -->
    <node pkg="mbot_gripper_controller_sim" type="sim_gripper_controller_node" name="gripper_interface_sim" output="screen" >
        <remap from="gripper_command" to="$(arg gripper_subcribe_command_topic)" />
        <remap from="gripper_actuation_sim" to="$(arg gripper_command_topic)" />

        <param name="gripper_configuration_open" type="double" value="$(arg gripper_configuration_open)" />
        <param name="gripper_configuration_close" type="double" value="$(arg gripper_configuration_close)" />
        <param name="gripper_joint_name" type="string" value="$(arg gripper_joint_name)" />
    </node>

</launch>
